### PR TITLE
Feature/android/auth/sso

### PR DESCRIPTION
**Pull request by timMikkelsen**

### CODE REVIEW

File: src/android/v3/guides/using-authentication-in-mapsindoors.md [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+```
+
+</mi-tab-panel>
+</mi-tabs>
+
+You will also need to provide a _redirect url_, but this is not provided by MapsIndoors. The callback url may be a [Android app link](https://developer.android.com/studio/write/app-link-indexing).
+
+> Note that the redirect link must be known to MapsIndoors and white-listed for your identity provider integration. You must inform us about all the links that you need for your application, both for development and production use so they can be white-listed.
+
+After you have made the request if using the library you will have to react on the recieved intent from the user loggin in.
```
Comment (fhansenMP) [inline src/android/v3/guides/using-authentication-in-mapsindoors.md:75]: spelling mistake: loggin -> logging

File: src/android/v3/guides/using-authentication-in-mapsindoors.md [hunk 2]
```diff
@@ -0,0 +1,166 @@
...
+            MapsIndoors.initialize(applicationContext, "apikey");
+        }
+    }
+}
+```
+
+</mi-tab-panel>
+</mi-tabs>
+
+The SDK will then make sure that all requests for data is performed using this access token.
```
Comment (fhansenMP) [inline src/android/v3/guides/using-authentication-in-mapsindoors.md:164]: The SDK will ensure all subsequent performed data requests will include the set access token.

File: src/android/v3/guides/using-authentication-in-mapsindoors.md [hunk 3]
```diff
@@ -0,0 +1,166 @@
...
+<mi-tabs>
+<mi-tab label="Java" tab-for="java"></mi-tab>
+<mi-tab label="Kotlin" tab-for="kotlin"></mi-tab>
+<mi-tab-panel id="java">
+
+```java
+private void checkApiKeyValidityAndInitializeSDK() {
+    MPApiKeyValidatorService.checkAuthToken("apikey", error -> {
+        if (error != null) {
+            //An error happened authentication was not succesful.
```
Comment (ChristianWolf-mapspeople) [inline src/android/v3/guides/using-authentication-in-mapsindoors.md:136]:
```suggestion
            //An error happened, authentication was not succesful.
```

File: src/android/v3/guides/using-authentication-in-mapsindoors.md [hunk 4]
```diff
@@ -0,0 +1,166 @@
...
+```
+
+</mi-tab-panel>
+<mi-tab-panel id="kotlin">
+
+```kotlin
+fun checkApiKeyValidityAndInitializeSDK() {
+    MPApiKeyValidatorService.checkAuthToken("apikey") {
+        if (it != null) {
+            //An error happened authentication was not succesful.
```
Comment (ChristianWolf-mapspeople) [inline src/android/v3/guides/using-authentication-in-mapsindoors.md:152]:
```suggestion
            //An error happened, authentication was not succesful.
```

**Review by ChristianWolf-mapspeople — CHANGES_REQUESTED** [review]

**Review by ChristianWolf-mapspeople — APPROVED** [review]

**Review by ChristianWolf-mapspeople — APPROVED** [review]

**Review by fhansenMP — APPROVED** [review]

**Pull request merged**